pkgs/applications/misc/gpsbabel/debian-patches.nix

```diff
@@ -0,0 +1,26 @@
+# Generated by debian-patches.sh from debian-patches.txt
```

Comment:
I didn't know about this script. Should we call it in the update script?

Reply:
That is https://github.com/NixOS/nixpkgs/blob/master/maintainers/scripts/debian-patches.sh.
No, it's not for update script, it should be regenerated only after manual editing of debian-patches.txt.